sdk/metric/meter_test.go

```diff
@@ -2064,3 +2068,208 @@ func TestHistogramBucketPrecedenceOrdering(t *testing.T) {
 		})
 	}
 }
+
+func TestObservableDropAggregation(t *testing.T) {
+	testcases := []struct {
+		name                   string
+		views                  []View
+		expectedObservableName []string
+		expectedLogs           []map[string]interface{}
+	}{
+		{
+			name:  "default",
+			views: []View{},
+			expectedObservableName: []string{
+				"observable.int64.counter",
```

Comment:
Can we declare all the observable names in a `const` section in the beginning of the test?

I think it will make the test readable and easier to navigate.

Something like

```go
const (
  intCounterName = "observable.int64.counter"
  ...
  dropIntCounterName = "unregistered.observable.int64.counter"
```